concepticondata/conceptlists/Anonby-2018-1500.tsv

```diff
@@ -0,0 +1,1501 @@
+Anonby-2018-1500-1147	1147	1342	SUMMER	summer	تابستان	tābestān	Environment	Weather	Seasons
+Anonby-2018-1500-1148	1148	1276	AUTUMN	autumn, fall	پاییز	pāyiz	Environment	Weather	Seasons
+Anonby-2018-1500-1149	1149	1471	WINTER	winter	زمستان	zemestān	Environment	Weather	Seasons
+Anonby-2018-1500-1150	1150	630	LIGHT (RADIATION)	light	نور	nur	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1151	1151	3276	SUNSHINE	sunshine	نورِ خورشید	nur-e xoršid	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1152	1152			moonlight	نورِ ماه	nur-e māh	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1153	1153	1388	SHADE	shadow	سایه	sāye	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1154	1154	11	DARKNESS	darkness	تاریکی	tāriki	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1155	1155	892	TIME	time	وقت، زمان	ʋaġt, zamān	Environment	Time	
+Anonby-2018-1500-1156	1156	1376	NOW	now	حالا، اکنون	hālā, aknun	Environment	Time	
+Anonby-2018-1500-1157	1157	1882	BEFORE	before	قبل، پیش از	ġabl, piš az	Environment	Time	
+Anonby-2018-1500-1158	1158	1685	AFTER	after	بعد، پس از	ba’d, pas az	Environment	Time	
+Anonby-2018-1500-1159	1159	672	EARLY	early	زود	zud	Environment	Time	
+Anonby-2018-1500-1160	1160	477	LATE	late	دیر	dir	Environment	Time	
+Anonby-2018-1500-1161	1161	374	ONE TIME	once	یک بار، یک باره 	yekbār, yekbāre	Environment	Time	
+Anonby-2018-1500-1162	1162	204	AGAIN	again	دوباره، باز	dobāre, bāz	Environment	Time	
+Anonby-2018-1500-1163	1163	1544	SOMETIMES	sometimes	بعضی اوقات	ba’zi aʋġāt	Environment	Time	
+Anonby-2018-1500-1164	1164	1852	OFTEN	often	اغلب	aġlab	Environment	Time	
+Anonby-2018-1500-1165	1165			usually	معمولاً	ma’mulan	Environment	Time	
+Anonby-2018-1500-1166	1166	1676	ALWAYS	always	همیشه	hamiše	Environment	Time	
+Anonby-2018-1500-1167	1167	1656	NEVER	never	هرگز، هیچ وقت	hargez, hičʋaġt 	Environment	Time	
+Anonby-2018-1500-1168	1168			spend time, pass time	وقت گذراندن	ʋaġt gozarāndan	Environment	Time	
+Anonby-2018-1500-1169	1169	1590	WAIT (FOR)	wait	انتظار کشیدن، منتظر بودن	entezār kešidan, montazer budan	Environment	Time	
+Anonby-2018-1500-1170	1170	1225	DAY (NOT NIGHT)	day	روز	ruz	Environment	Time	Time periods
+Anonby-2018-1500-1171	1171	1370	MONTH	month	ماه	māh	Environment	Time	Time periods
+Anonby-2018-1500-1172	1172	1226	YEAR	year	سال	sāl	Environment	Time	Time periods
+Anonby-2018-1500-1173	1173	1283	TODAY	today	امروز	emruz	Environment	Time	Time periods
+Anonby-2018-1500-1174	1174	1174	YESTERDAY	yesterday	دیروز	diruz	Environment	Time	Time periods
+Anonby-2018-1500-1175	1175	1180	DAY BEFORE YESTERDAY	day before yesterday	پریروز	pariruz	Environment	Time	Time periods
+Anonby-2018-1500-1176	1176	1329	TOMORROW	tomorrow	فردا	fardā	Environment	Time	Time periods
+Anonby-2018-1500-1177	1177	1179	DAY AFTER TOMORROW	day after tomorrow	پس فردا 	pasfardā	Environment	Time	Time periods
+Anonby-2018-1500-1178	1178	2914	ONCE (IN THE PAST)	olden times	زمانِ قدیم	zamān-e ġadim	Environment	Time	Time periods
```

Comment:
I would rather leave it unmapped. Olden times refers to a rather big stretch of time while ONCE (IN THE PAST) refers to a certain point in the past.